common/interpreter/wawaka_wasm/CMakeLists.txt

```
@@ -40,137 +40,127 @@ SET(WASM_MEM_CONFIG "$ENV{WASM_MEM_CONFIG}")
 SET(PDO_INTERPRETER "$ENV{PDO_INTERPRETER}")
 
 # Reset default linker flags
-set (CMAKE_SHARED_LIBRARY_LINK_C_FLAGS "")
-set (CMAKE_SHARED_LIBRARY_LINK_CXX_FLAGS "")
+SET (CMAKE_SHARED_LIBRARY_LINK_C_FLAGS "")
+SET (CMAKE_SHARED_LIBRARY_LINK_CXX_FLAGS "")
 
 # Set WAMR_BUILD_TARGET
-if (NOT DEFINED WAMR_BUILD_TARGET)
-  if (CMAKE_SIZEOF_VOID_P EQUAL 8)
+IF (NOT DEFINED WAMR_BUILD_TARGET)
+  IF (CMAKE_SIZEOF_VOID_P EQUAL 8)
     # Build as X86_64 by default in 64-bit platform
-    set (WAMR_BUILD_TARGET "X86_64")
-  else ()
+    SET (WAMR_BUILD_TARGET "X86_64")
+  ELSE ()
     # Build as X86_32 by default in 32-bit platform
-    set (WAMR_BUILD_TARGET "X86_32")
-  endif ()
-endif ()
+    SET (WAMR_BUILD_TARGET "X86_32")
+  ENDIF ()
+ENDIF ()
 
 if (NOT CMAKE_BUILD_TYPE)
-  set(CMAKE_BUILD_TYPE Release)
+  SET(CMAKE_BUILD_TYPE Release)
 endif ()
```

A few more nitpicky changes for consistency.

```suggestion
ENDIF()
```